From 58efd3c61c52e860c38b303067648a2b417ba1a8 Mon Sep 17 00:00:00 2001
From: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
Date: Thu, 24 May 2018 04:39:59 -0500
Subject: [PATCH] Makefile: add a target to build sbl hypervisor

To build with only one command the sbl and efi hypervisor a target was
added which build the SBL hypervisor version.

Now you can call "make all sbl-hypervisor" and
"make install sbl-hypervisor-install"

Signed-off-by: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
---
 Makefile | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/Makefile b/Makefile
index 3d7f25a..e2477cd 100644
--- a/Makefile
+++ b/Makefile
@@ -17,6 +17,11 @@ hypervisor:
 	make -C $(T)/hypervisor HV_OBJDIR=$(HV_OUT) PLATFORM=$(PLATFORM) RELEASE=$(RELEASE) clean
 	make -C $(T)/hypervisor HV_OBJDIR=$(HV_OUT) PLATFORM=$(PLATFORM) RELEASE=$(RELEASE)
 
+sbl-hypervisor:
+	@mkdir -p $(HV_OUT)-sbl
+	make -C $(T)/hypervisor HV_OBJDIR=$(HV_OUT)-sbl PLATFORM=sbl RELEASE=$(RELEASE) clean
+	make -C $(T)/hypervisor HV_OBJDIR=$(HV_OUT)-sbl PLATFORM=sbl RELEASE=$(RELEASE)
+
 devicemodel:
 	make -C $(T)/devicemodel DM_OBJDIR=$(DM_OUT) clean
 	make -C $(T)/devicemodel DM_OBJDIR=$(DM_OUT)
@@ -38,6 +43,9 @@ install: hypervisor-install devicemodel-install tools-install
 hypervisor-install:
 	make -C $(T)/hypervisor HV_OBJDIR=$(HV_OUT) PLATFORM=$(PLATFORM) RELEASE=$(RELEASE) install
 
+sbl-hypervisor-install:
+	make -C $(T)/hypervisor HV_OBJDIR=$(HV_OUT)-sbl PLATFORM=sbl RELEASE=$(RELEASE) install
+
 devicemodel-install:
 	make -C $(T)/devicemodel DM_OBJDIR=$(DM_OUT) install
 
-- 
2.17.0

